src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,37 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { FullTodo } from '../../types/FullTodo';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
```

Comment:
Consider moving the CSS import to a separate file or module for better separation of concerns. 💅